internal/registry/credentialrequest/rest_test.go

```diff
@@ -27,14 +27,27 @@ import (
 	"go.pinniped.dev/internal/testutil"
 )
 
+func TestRestMethods(t *testing.T) {
+	t.Parallel()
+	type storage interface {
+		rest.Creater
+		rest.NamespaceScopedStrategy
+		rest.Scoper
+		rest.Storage
+	}
+	require.Implements(t, (*storage)(nil), new(REST))
```

Comment:
Why a test over the traditional `var _ storage = &REST{}` to get a compile time error?

Reply:
I could go either way on this, but it's a marginally nicer error message.